spec/ADAS/ADAS.vspec

```diff
@@ -131,6 +145,28 @@ ObstacleDetection.IsError:
   type: sensor
   description: Indicates if obstacle sensor system incurred an error condition. True = Error. False = No Error.
 
+ObstacleDetection.Distance:
+  datatype: float
+  type: sensor
+  unit: m
+  description: Distance in meters to detected object
+
+ObstacleDetection.TimeGap:
+  datatype: int32
```

Comment:
Has negative number a meaning, or would uint32 suffice?
Has this one and `Distance` only significance if `ObstacleDetection.IsWarning` is True, or what values should be used if there is no obstacle?

Reply:
Unsigned is fine but wanted decimal instead of rounding to full integer.